src/schemas/table_schemas/guide_rna_sequences.json

```diff
@@ -0,0 +1,123 @@
+            "type": "string",
+            "description": "sequence of protospacer",
+            "constraints": {
+                "required": true
+            }
+        },
+        {
+            "name": "targeting",
+            "type": "boolean",
+            "description": "'TRUE' or 'FALSE', depending on whether or not the guide is targeting a locus of the genome relevant to the experiment. If guide is safe-targeting or non-targeting, targeting = False. 'safe-targeting' refers to guides designed to target genomic sites with no annotated function, used as negative controls, if targeting == False, guide_chr, guide_start, guide_end, strand, and pam can be set to NaN",
+            "constraints": {
+                "required": true
+            }
+        },
+        {
+            "name": "type",
+            "type": "string",
+            "description": "flexible tag describing the role of the guide in the experiment",
+            "constraints": {
+                "required": true,
+                "enum": [
+                    "safe-targeting",
+                    "non-targeting",
+                    "targeting",
+                    "PosControl",
+                    "NegControl",
+                    "Variant"
+                ]
+            }
+        },
+        {
+            "name": "guide_chr",
+            "type": "string",
+            "description": "chromosome of guide, ex. chrX, chr2, chr10"
+        },
+        {
+            "name": "guide_start",
+            "type": "integer",
+            "description": "base start position of guide",
+            "constraints": {
+                "minimum": 0
+            }
+        },
+        {
+            "name": "guide_end",
+            "type": "integer",
+            "description": "base end position of guide",
+            "constraints": {
+                "minimum": 0
+            }
+        },
+        {
+            "name": "strand",
+            "type": "string",
+            "description": "Strand of guide, + or -. If strandedness is unclear or unnecessary, use 'NaN' for the strand",
+            "constraints": {
+                "enum": [
+                    "+",
+                    "-"
+                ]
+            }
+        },
+        {
+            "name": "pam",
+            "type": "string",
+            "description": "protospacer adjacent motif"
+        },
+        {
+            "name": "(genomic_element)",
+            "type": "string",
+            "description": "type of the genomic element being targeted, ex. gene, enhancer, promoter, insulator"
+        },
+        {
+            "name": "(intended_target_name)",
+            "type": "string",
+            "description": "name of target element, should use existing conventions whenever possible: SPDI_id, ensembl_id, HGNC_id. If intended_target_name == NaN, then the target element will be referenced using its coordinates"
+        },
+        {
+            "name": "(intended_target_chr)",
```

Comment:
I dont think they meant it to be in parenthesis (they just wanted to indicate the optional elements)